internal/metrics/metricsManager.go

```diff
@@ -282,6 +282,10 @@ func NewMetricsManager(registry prometheus.Registerer, eventsHandler eventsapi.H
 		}, []string{hosts}),
 	}
 
+	for _, directory := range directoryUsageMonitorDirs {
```

Comment:
I think this job of looping with dirs can be done by the collector directly, WDYT?

Reply:
The advantage would be not registering potentially many collectors